cmd/containerd-stress/density.go

```diff
@@ -206,21 +206,21 @@ func parseStat(data string) (stat Stat, err error) {
 		return stat, fmt.Errorf("invalid stat data: %q", data)
 	}
 
-	parts := strings.SplitN(data[:i], "(", 2)
-	if len(parts) != 2 {
+	name, val, ok := strings.Cut(data[:i], "(")
+	if !ok {
 		return stat, fmt.Errorf("invalid stat data: %q", data)
 	}
 
-	stat.Name = parts[1]
-	_, err = fmt.Sscanf(parts[0], "%d", &stat.PID)
+	stat.Name = name
```

Comment:
`parts[1]` -> `name` here and then `parts[0]` -> `val`, something is backwards, was it incorrect previously?

Reply:
Good catch. I don't think I changed that one intentionally 🤔🙀

Would be fun though if it unintentionally fixed a bug 😂

I'm not at my computer right now to view it in within context, but will look later (of course feel free to beat me to that)

Reply:
Updated 👍 

I may have a look at updating this function to the latest one from runc, which received a bunch of fixes and improvements; https://github.com/opencontainers/runc/pull/2696

I can either use it from there, or update the fork we have here (will have a look what works best)